_posts/en/newsletters/2020-11-25-newsletter.md

```diff
@@ -0,0 +1,120 @@
+  mining pool [Poolin][] has created [a website][taprootactivation.com]
+  to help track miner support for activating
+  [taproot][topic taproot] (including [schnorr signatures][topic schnorr
+  signatures] and [tapscript][topic tapscript]).  As of this writing,
+  all pools listed on the site---who represent more than half of the
+  current network hash rate---have indicated that they will support
+  activation.  If this is reflective of the opinions of the remaining
+  pools, then it should be easy to accomplish the activation of taproot
+  after its implementation has been released and adopted by a moderate
+  number of users.  For additional information about the creation of the
+  website, see Bitcoin Magazine's [article][btcmag ta.com].
+
+## Selected Q&A from Bitcoin StackExchange
+
+*[Bitcoin StackExchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments to help curious or confused users.  In
+this monthly feature, we highlight some of the top-voted questions and
+answers posted since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+FIXME:bitschmidty
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+FIXME:harding to update on Tuesday
+
+- [C-Lightning 0.9.2rc1][C-Lightning 0.9.2] is a release candidate for
+  the next maintenance version of C-Lightning.  It contains new
+  features, updated options, and bug fixes.
+
+- [Bitcoin Core 0.21.0rc1][Bitcoin Core 0.21.0] is a release candidate
+  for the next major version of this full node implementation and its
+  associated wallet and other software.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1 repo],
+[Hardware Wallet Interface (HWI)][hwi repo], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+*Note: some commits to Bitcoin Core mentioned below apply to its
+development branch and so those changes will likely not be released
+until version 0.22, about six months after the release of the upcoming
+version 0.21.*
+
+- [Bitcoin Core #20305][] begins standardizing RPC fee rate units to satoshis
+  per vbyte (sat/vB), marking a shift away from BTC per 1,000 vbytes (BTC/kvB), by introducing an
+  `fee_rate` parameter to the `sendtoaddress`,
+  `sendmany`, `fundrawtransaction`, and `walletcreatefundedpsbt` RPCs as well as
+  to the experimental new `send` RPC. In addition, the `bumpfee` RPC `fee_rate`
```

Comment:
maybe link the experimental new send RPC to https://bitcoinops.org/en/newsletters/2020/09/23/#bitcoin-core-16378